package.json

```diff
@@ -48,6 +49,7 @@
     "babel-loader": "6.2.10",
     "babel-plugin-transform-object-rest-spread": "6.20.2",
     "babel-preset-es2015": "6.18.0",
+    "chrome-launch": "1.1.4",
```

Comment:
Ahhh I wasn't aware of the issue with the shrinkwrap. Declaring the dependencies twice doesn't sound right either. Let's continue with the good old way 😉